src/pages/sighting/statusCard/StatusCard.jsx

```diff
@@ -265,14 +298,21 @@ export default function StatusCard({ sightingData }) {
         <TimelineStep
           Icon={IdentificationIcon}
           titleId="IDENTIFICATION"
+          isUserInTheLoop={false}
```

Comment:
Why not add a default of false in the TimelineStep component?

Reply:
This prop was removed in https://github.com/WildMeOrg/codex-frontend/pull/432/commits/3ae2ef46f56ba96e3604684f5edffbffb7314b66.